### ISSUE TITLE
"closable" AC power warning is in a confusing location

### ISSUE THREAD

**Issue by superm1** [canonical/firmware-updater#215]
When I run `Firmware Updater` without an AC adapter plugged in I get a warning in the top of the window that some stuff can't be updated without AC.

I think the warning itself is good to have, but I think having an X for the warning in the same place that you would usually "close the window" is confusing.  I think it would be better to put this somewhere else prominent; like the bottom of the window.

![image](https://github.com/canonical/firmware-updater/assets/494526/d289056d-cea0-46b6-95c4-1b7a3bea2355)

**Comment by Glen-Campbell-Bose** [comment]
I agree with this and have put together some code to make the warning message into an AlertDialog. Is that acceptable? 
There's another case where if the BIOS boot mode is "Legacy", you can't update. I've created an AlertDialog for this as well. Is that a separate issue, or can I include that with this issue?

![Screenshot from 2024-01-16 17-44-49](https://github.com/canonical/firmware-updater/assets/22241543/bc6fbbd5-749c-4cb7-b2c2-99b688c9a503)

**Comment by superm1** [comment]
That looks great to me!

**Comment by superm1** [comment]
IMO For BIOS mode of legacy I think it should have an option to waive in the future/ignore it.  That can be expected for some people and there isn't much they can do.

**Comment by Glen-Campbell-Bose** [comment]
That's a good idea. A little check-box on the warning "don't show this in the future"?
Although I tripped over this and it motivated me to reinstall everything as UEFI-bootable.
It was a pain in the neck, but I was glad I did it.

**Comment by superm1** [comment]
> That's a good idea. A little check-box on the warning "don't show this in the future"?
> Although I tripped over this and it motivated me to reinstall everything as UEFI-bootable.
> It was a pain in the neck, but I was glad I did it.

Yeah something like that.

There's other things that could be useful to prompt and save a setting like this in the future too so I suggest building it into a reusable snippet or function.

The other thing that comes to mind is opting in to reports after an update.

**Comment by Glen-Campbell-Bose** [comment]
OK.
